### DISCUSSION TITLE
reduce line spacing between publications title, authors and buttons?

### DISCUSSION THREAD

**Discussion by p-costa** [alshedivat/al-folio#303] [Q&A] [answered]
Hi all,

Thanks for this awesome template :). Do you know how I can reduce the line spacing between the publication title, author names, and buttons? It seems a bit too large for my taste now.

Thank you in advance!
Pedro

![image](https://user-images.githubusercontent.com/29615530/120931780-a484b080-c6e2-11eb-8052-2881ad2cd7c4.png)

**Comment by XuhuiZhou** [comment]
I am also wondering that. It seems many other folks' pages using this template have already reduced the space. Will post here if I find a solution :)

**Reply by mbarbetti** [reply]
To adjust whatever related with page format you should modify the file `_base.scss` in `_sass`. More precisely, you should look for the class `.publications` and choose the preferred values for `margin-top` and `margin-bottom` options. Notice that these options refer to the particular tag content where are declared!

**Comment by alshedivat** [comment, marked as answer]
The issue should be fixed by 63f278ac927293b0fbf82d268a114d4df209953e.
